### DISCUSSION TITLE
QualCoder 2.9

### DISCUSSION THREAD

**Discussion by ccbogel** [ccbogel/QualCoder#604] [Announcements]
To update release notes soon.

<hr /><em>This discussion was created from the release <a href='https://github.com/ccbogel/QualCoder/releases/tag/'>QualCoder 2.9</a>.</em>

**Comment by Flooosh** [comment]
I tried to install 2.9 on Linux Mint 20.3 (based on Ubuntu 20.04) but it doesn't work. Error message says: File "QualCoder-2.9-Ubuntu", line 1
SyntaxError: Non-UTF-8 code starting with '\xaf' in file QualCoder-2.9-Ubuntu on line 2, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

**Comment by ccbogel** [comment]
Hmm. Ok. It worked fine on my ubuntu 20.04. Not sure why. 
Csn you install the long way, by following the read me instructions,  installing python modules etc from the terminal
